docs/source/guide/scim_setup.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Set up SCIM2 for Label Studio
+short: Set up SCIM2
+badge: <i class='ent'></i>
+type: guide
+order: 253
+meta_title: System for Cross-domain Identity Management (SCIM) for Label Studio Enterprise
+meta_description: Label Studio Enterprise documentation for setting up SCIM2
+---
+
+System for Cross-domain Identity Management (SCIM) is a popular protocol to manage access for services and applications across an organization. This guide helps to set up SCIM integration to manage access to Label Studio Enterprise in your organization. 
+
+Using SCIM provider, you can manage access to Label Studio Enterprise workspaces, grant roles to individual users and groups.
+
+## Requirements
+
+SCIM interacts with your SSO integration. 
+
+> Note: 
+Okta or similar SSO providers have SCIM integration based on SSO.
+
+If you do not have SSO set up yet, then follow [Set up SSO](auth_setup.md).
```

Comment:
!!! attention "important"
     If you do not have SSO set up yet, then follow [Set up SSO](auth_setup.md).